packages/zowe-explorer/src/job/actions.ts

```diff
@@ -549,16 +549,16 @@ export async function filterJobs(jobsProvider: IZoweTree<IZoweJobTreeNode>): Pro
     for (const level of jobsProvider.mSessionNodes) {
         if (level.label === "zosmf") {
```

Comment:
We don't want to have any particular type calls like checking for zosmf profile type or if this is a check for profile name.  We want to be able to handle this for all profile types and names.  This may need to be a check on context value for if session node.